test/TestAccountHelper.js

```diff
@@ -0,0 +1,219 @@
+const nearAPI = require('near-api-js');
+
+const constants = require('../constants');
+
+const { SERVER_EVENTS } = constants;
+
+function extractValueFromHash(hash, key) {
```

Comment:
nit: given this is only used one time and essentially 2-liner, I'd just inline it.

Reply:
P.S. Wonder why you call it hash, this brings me back to the time I was writing some hacky Perl in one of my first web apps :)

Reply:
:+1: I guess it could have been `extractValueFromObj()` :). Dunno why my mind wrote it that way... :) In-lining.